ang/crmUi.js

```diff
@@ -1002,16 +1002,22 @@
         scope: {
           crmDocumentTitle: '='
         },
+        limit: 'ae',
         link: function(scope, $el, attrs) {
           function update() {
             $timeout(function() {
               var newPageTitle = _.trim($el.html()),
-                newDocumentTitle = scope.crmDocumentTitle || $el.text();
+                newDocumentTitle = scope.crmDocumentTitle || $el.text(),
+                cls = $el.attr('class').split(' '),
+                classes = _.filter(cls, function(c) {
+                  return c.indexOf('ng-') !== 0;
```

Comment:
This seems to say "sync the CSS classes, except when you shouldn't, and here's a rule-of-thumb about when you shouldn't". How confident are you in this rule-of-thumb? It seems like it would transfer over bits for other directives. (For example, `crm-page-title` is replicated over.)

I don't want to block if you feel strongly about it, but it seems like it could just as easily list the sync-able classes in different attribute with less risk of conflicts, eg:

```html
<h1 crm-page-title crm-page-class="foo bar">{{ts('Hello')}}</h1>
```

then change `$el.attr('class')` to `scope.crmPageClass`, and you don't the filter.